pkg/rancher-desktop/utils/_demo_marketplace_items.js

```diff
@@ -1,7 +1,7 @@
 export const demoMarketplace = {
   summaries: [
     {
-      id:                 '',
+      id:                 'epinio',
```

Comment:
Huh, I didn't think the `id` here was relevant. (`extension.id` above is actually the extension image, i.e. `slug` + tag.) I guess it doesn't hurt?

Reply:
It's not relevant, I reverted it.